egs/aspire/s5/local/multi_condition/aspire_prep_rir_noise_list.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to generate rir_list and noise_list in aspire
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import argparse, glob, math, os, sys
+
+
+def GetArgs():
+    parser = argparse.ArgumentParser(description="Prepare rir_list and noise_list for Aspire  "
+                                                 "Usage: reverberate_data_dir.py [options...] <in-data-dir> <out-data-dir> "
+                                                 "E.g. reverberate_data_dir.py "
+                                                 "data/impulses_noises data/impulses_noises/info",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("input_dir", help="Input data directory")
+    parser.add_argument("output_dir", help="Output data directory")
+    print(' '.join(sys.argv))
+    args = parser.parse_args()
+
+    return args
+
+
+# This function generate the rir_list file for the aspire real RIR
+def GenerateRirListFile(input_dir, output_dir):
+  rir_list_file = open(output_dir + "/rir_list", 'w')
+  rir_id = 1
+  room_id = 1
+  for db in ["RVB2014", "RWCP", "air"]:
+    rir_files = glob.glob(input_dir + "/{0}_*.wav".format(db))
+    for rir in rir_files:
+      filename = rir.split('/')[-1]
+      if "noise" not in filename:
+        rir_list_file.write('--rir-id {0} --room-id {1} {2}\n'.format(str(rir_id).zfill(5), str(room_id).zfill(3), rir))
+        rir_id += 1
+    room_id += 1
+  rir_list_file.close()
+
+
+# This function generate the noise_list file from the aspire noise-rir pair 
+def GenerateNoiseListFile(input_dir, output_dir):
```

Comment:
There is a `data/impulses_noises/info` which has all the information you need.  It defines different rooms and the noises and impulse responses available for that room (e.g. `data/impulses_noises/info/noise_impulse_RVB2014_largeroom1`).

Please do not assign the same `room_id` to all RIRs in a given database. This is not the right usage of `room_id` field.